megaqc/settings.py

```diff
@@ -69,6 +71,7 @@ class DevConfig(Config):
     DEBUG_TB_ENABLED = True
     CACHE_TYPE = 'simple'  # Can be "memcached", "redis", etc.
     WTF_CSRF_ENABLED = False  # Allows form testing
+    SQLALCHEMY_RECORD_QUERIES = True
```

Comment:
What does this do?

Reply:
When running in debug mode, it keeps track of queries in the sidebar thingy. I enabled it when I was hunting bugs, figured it'd be alright for future debug activities. Can easily be removed if not.

Reply:
Super cool 👍 We previously had log statements pushing these to the terminal. Having it front end is much nicer.